apps/person.py

```diff
@@ -0,0 +1,16 @@
+class Person(object):
+    def __init__(self, person_name):
+        self.person_name = person_name
```

Comment:
What about other attributes of class person i.e person_attributes(fellow, staff)